<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Idas-MacBook-Air.local" name="se.nrm.dina.data.exceptions.DinaExceptionNGTest" tests="4" failures="0" timestamp="10 Feb 2016 16:35:32 GMT" time="0.028" errors="0">
  <testcase name="testDinaExceptionConstructWithMsg" time="0.000" classname="se.nrm.dina.data.exceptions.DinaExceptionNGTest"/>
  <testcase name="testGetErrorCode" time="0.001" classname="se.nrm.dina.data.exceptions.DinaExceptionNGTest"/>
  <testcase name="testDinaExceptionConstructWithMsgAndErrorCode" time="0.000" classname="se.nrm.dina.data.exceptions.DinaExceptionNGTest"/>
  <testcase name="testDinaExceptionConstruct" time="0.027" classname="se.nrm.dina.data.exceptions.DinaExceptionNGTest"/>
</testsuite> <!-- se.nrm.dina.data.exceptions.DinaExceptionNGTest -->
